packages/flutter/test/widgets/semantics_test.dart

```diff
@@ -765,6 +765,45 @@ void main() {
     semantics.dispose();
   });
 
+  testWidgets('MergeSemantics merges CustomSemanticsActions from its children', (
+    WidgetTester tester,
+  ) async {
+    final SemanticsHandle semantics = tester.ensureSemantics();
+
+    await tester.pumpWidget(
+      MergeSemantics(
+        child: Column(
+          children: <Widget>[
+            Semantics(
```

Comment:
To properly test this , they have to create their own node, so I think we need container:true for both semantics widget

Reply:
I see. Thanks! Just updated!